Formula/luvit.rb

```diff
@@ -47,6 +52,21 @@ class Luvit < Formula
       url "https://github.com/luvit/luvi/commit/b2e501deb407c44a9a3e7f4d8e4b5dc500e7a196.patch?full_index=1"
       sha256 "be3315f7cf8a9e43f1db39d0ef55698f09e871bea0f508774d0135c6375f4291"
     end
+
+    livecheck do
+      url "https://raw.githubusercontent.com/luvit/luvit/#{LATEST_VERSION}/Makefile"
+      regex(/LIT_VERSION=["']?(\d+(?:\.\d+)+)["']?$/i)
+      strategy :page_match do |page, regex|
+        lit_version = page.match(regex)[1]
+        next if lit_version.blank?
+
+        get_lit_url = "https://raw.githubusercontent.com/luvit/lit/#{lit_version}/get-lit.sh"
+        get_lit_page = Homebrew::Livecheck::Strategy.page_content(get_lit_url)
+        next if get_lit_page[:content].blank?
+
+        get_lit_page[:content].match(/LUVI_VERSION:-v?(\d+(?:\.\d+)+)/)[1]
```

Comment:
```suggestion
        lit_version = page[regex, 1]
        next if lit_version.blank?

        get_lit_page = Homebrew::Livecheck::Strategy.page_content(
          "https://raw.githubusercontent.com/luvit/lit/#{lit_version}/get-lit.sh",
        )
        next if get_lit_page[:content].blank?

        get_lit_page[:content][/LUVI_VERSION:-v?(\d+(?:\.\d+)+)/i, 1]
```

Just a few small tweaks and this should be good to go.